### PR TITLE
fix: update asc and add asconfig.json

### PR DESCRIPTION
**Pull request by willemneal**
Hopefully updating asc won't cause unforeseen issues.

### CODE REVIEW

**Comment by willemneal** [comment]
No problem!

File: stdlib/asconfig.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "options": {
+    "-O3z": true,
```
Comment (MaxGraey) [inline stdlib/asconfig.json:3]: Btw it was `-O3` before. It's could significant affected on performance for `memory.copy` / `memory.compare`

Comment (ospencer) [reply stdlib/asconfig.json:3]: Ah, good catch.

Comment (willemneal) [reply stdlib/asconfig.json:3]: @MaxGraey How so?  Is this a binaryen issue?

Comment (MaxGraey) [reply stdlib/asconfig.json:3]: > How so?

AssemblyScript uses conditionally compiled branches specialized for speed and / or size. Sometimes for good size reduction we use compleately different implementations.

>  Is this a binaryen issue?

Binaryen's also sensitive to level of shrinkage. Especially on inlining

**Pull request merged**